src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -177,11 +175,7 @@ private string ConvertToPrettyJsonString(string json)
         /// <returns></returns>
         private int ConvertList(string json, int index, StringBuilder result, string padString, int numberOfSpaces)
         {
-            if (_stopping)
-            {
-                throw new StopUpstreamCommandsException(this);
-            }
-
+            CheckStopping();
```

Comment:
I think this should be either inside in for loop or mirrored there.

Reply:
The only potential expensive calls in that loop are ConvertDictionary() and ConvertList() which already handles stopping now